app/helpers/claims_helper.rb

```diff
@@ -28,9 +28,9 @@ def claim_answers(claim)
   def identity_answers(claim)
     [
       ["Full name", claim.full_name, "full-name"],
-      ["Address", claim.address, "address"],
-      ["Date of birth", l(claim.date_of_birth), "date-of-birth"],
-      ["Teacher reference number", claim.teacher_reference_number, "teacher-reference-number"],
+      ["What is your address?", claim.address, "address"],
```

Comment:
We should probably be consistent with using statements or questions:

https://www.gov.uk/service-manual/design/form-structure#structure-your-form-to-help-users

Reply:
That's a good shout Robbie I'll run through and phrase everything as a question. 👍